src/components/Tabs/TabList/TabList.scss

```diff
@@ -26,10 +26,10 @@
     .tabs-list {
       width: 100%;
 
-      .tab--wrapper {
+      .tab--wrapper-override {
```

Comment:
why ?

Reply:
wanted to rename it somehow to show that it's styles which overrides `tab--wrapper` through Tab props, not through global class anymore
Can be renamed to `&_tab--wrapper` e.g. if u prefer. Or what r u proposing here?

Reply:
oh, so this is the internal conflicts that we had ?

wrapper feels better